V4.1

henkkelder opened the pull request:
I've got the example working. Seems fine on PC. Gives CORS errors op my Android device.

ihadeed commented on .vs/VSWorkspaceState.json:
```diff
@@ -0,0 +1,8 @@
+{
```
Could you please remove the .vs directory and/or add it to .gitignore

henkkelder replied:
I did. 
I had to make more changes to get it working on my Android & IOS devices.
Please check my commits. I added comment when I though it was useful

henkkelder commented:
The v4 branch is a sample project. How do we convert this to a usable component? Or do I not understand how it works?

ihadeed commented:
@henkkelder it's now published under `ionic-image-loader@beta`

henkkelder commented:
Nice. I will test it today.

 

From: Ibby Hadeed <notifications@github.com> 
Sent: vrijdag 14 december 2018 11:17
To: zyra/ionic-image-loader <ionic-image-loader@noreply.github.com>
Cc: Henk Kelder <henk.kelder@solcon.nl>; Mention <mention@noreply.github.com>
Subject: Re: [zyra/ionic-image-loader] V4.1 (#224)

 

@henkkelder <https://github.com/henkkelder>  it's now published under ionic-image-loader@beta

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/zyra/ionic-image-loader/pull/224#issuecomment-447280469> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AIufNhhEmi6ENlEYItNPM3WItYamq1ifks5u43q3gaJpZM4Y6ES3> .  <https://github.com/notifications/beacon/AIufNhzbHbpiLHALobdC24464EI-P8lUks5u43q3gaJpZM4Y6ES3.gif>

henkkelder commented:
How do I use the ConfigService? This import doesn’t seem to work:

 

import { ImageLoaderConfigService } from "ionic-image-loader";

 

 

From: Ibby Hadeed <notifications@github.com> 
Sent: vrijdag 14 december 2018 11:17
To: zyra/ionic-image-loader <ionic-image-loader@noreply.github.com>
Cc: Henk Kelder <henk.kelder@solcon.nl>; Mention <mention@noreply.github.com>
Subject: Re: [zyra/ionic-image-loader] V4.1 (#224)

 

@henkkelder <https://github.com/henkkelder>  it's now published under ionic-image-loader@beta

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/zyra/ionic-image-loader/pull/224#issuecomment-447280469> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AIufNhhEmi6ENlEYItNPM3WItYamq1ifks5u43q3gaJpZM4Y6ES3> .  <https://github.com/notifications/beacon/AIufNhzbHbpiLHALobdC24464EI-P8lUks5u43q3gaJpZM4Y6ES3.gif>

henkkelder commented:
Did you see me previous question?

 

From: Henk Kelder <henk.kelder@solcon.nl> 
Sent: vrijdag 14 december 2018 12:03
To: 'zyra/ionic-image-loader' <reply@reply.github.com>
Subject: RE: [zyra/ionic-image-loader] V4.1 (#224)

 

How do I use the ConfigService? This import doesn’t seem to work:

 

import { ImageLoaderConfigService } from "ionic-image-loader";

 

 

From: Ibby Hadeed <notifications@github.com <mailto:notifications@github.com> > 
Sent: vrijdag 14 december 2018 11:17
To: zyra/ionic-image-loader <ionic-image-loader@noreply.github.com <mailto:ionic-image-loader@noreply.github.com> >
Cc: Henk Kelder <henk.kelder@solcon.nl <mailto:henk.kelder@solcon.nl> >; Mention <mention@noreply.github.com <mailto:mention@noreply.github.com> >
Subject: Re: [zyra/ionic-image-loader] V4.1 (#224)

 

@henkkelder <https://github.com/henkkelder>  it's now published under ionic-image-loader@beta

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/zyra/ionic-image-loader/pull/224#issuecomment-447280469> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AIufNhhEmi6ENlEYItNPM3WItYamq1ifks5u43q3gaJpZM4Y6ES3> .  <https://github.com/notifications/beacon/AIufNhzbHbpiLHALobdC24464EI-P8lUks5u43q3gaJpZM4Y6ES3.gif>

ihadeed commented:
@henkkelder sorry for the late response.

I just fixed it and published `ionic-image-loader@7.0.0-beta.1`.

Thank you for your help

henkkelder commented:
Modified my code to use the corrected beta. 

Seems to work fine.

 

Groeten,

Henk Kelder

 

 

From: Ibby Hadeed <notifications@github.com> 
Sent: vrijdag 21 december 2018 15:02
To: zyra/ionic-image-loader <ionic-image-loader@noreply.github.com>
Cc: Henk Kelder <henk.kelder@solcon.nl>; Mention <mention@noreply.github.com>
Subject: Re: [zyra/ionic-image-loader] V4.1 (#224)

 

@henkkelder <https://github.com/henkkelder>  sorry for the late response.

I just fixed it and published ionic-image-loader@7.0.0-beta.1 <mailto:ionic-image-loader@7.0.0-beta.1> .

Thank you for your help

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/zyra/ionic-image-loader/pull/224#issuecomment-449396277> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AIufNpcgzZmm5c4CLejWdk62RC8GWbsYks5u7OnlgaJpZM4Y6ES3> .  <https://github.com/notifications/beacon/AIufNpr0iaJwH6f8D0aJq5PDG__nBd91ks5u7OnlgaJpZM4Y6ES3.gif>

henkkelder commented:
Happy newyear Ibby,

 

Could you please have a look at my PR for V4.

 

Regards,

Henk Kelder from the Netherlands.

 

From: Ibby Hadeed <notifications@github.com> 
Sent: vrijdag 14 december 2018 11:17
To: zyra/ionic-image-loader <ionic-image-loader@noreply.github.com>
Cc: Henk Kelder <henk.kelder@solcon.nl>; Mention <mention@noreply.github.com>
Subject: Re: [zyra/ionic-image-loader] V4.1 (#224)

 

@henkkelder <https://github.com/henkkelder>  it's now published under ionic-image-loader@beta

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/zyra/ionic-image-loader/pull/224#issuecomment-447280469> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AIufNhhEmi6ENlEYItNPM3WItYamq1ifks5u43q3gaJpZM4Y6ES3> .  <https://github.com/notifications/beacon/AIufNhzbHbpiLHALobdC24464EI-P8lUks5u43q3gaJpZM4Y6ES3.gif>
